examples/component/test/example.test.js

```diff
@@ -0,0 +1,43 @@
+import { expect } from 'chai';
+import { getComponentOutput } from 'astro-component-tester';
```

Comment:
I'm not sure on having this inside this template. Both `astro-component-tester` and a test setup. I agree that it's useful - but it feels weirdly convoluted and opinionated for a general template. I think I'd remove all of it tbh

Reply:
Removed, thanks